docs/index.md

```diff
@@ -32,13 +32,29 @@ cp config.ini.template config.ini
 
 The last step, running web.py is optional if you don not want to deploy the web visual interface. The ports to run the app are 90100 for the API and 5000 for the web interface. They can be configured if needed.
 
+
+
 ### Docker version deployment
 
 To deploy a docker container, just run the latest image:
 
 ```
 docker run --name=fair_eva -p 9090:9090 -p 5000:5000 -dit --network host
 ```
+## Example with fair.py 
```

Comment:
The content of this section should be moved to the "Getting started" section. Here there is already a list of terminal commands to quick start the FAIR API and Web. The text you are adding here complements what is being written there with a concrete example on how to make a request to the API.

Reply:
¿So it should be only in the getting started? OK

Reply:
Moved